### DISCUSSION TITLE
game wont open

### DISCUSSION THREAD

**Discussion by monikaheart** [Monika-After-Story/MonikaModDev#9167] [QA - tech support + general] [answered]
Please help, I literally have no idea what this means.
![image](https://user-images.githubusercontent.com/100363060/155543449-baab90b5-1527-46e5-9c46-e9c9ff7a0455.png)

**Comment by ThePotatoGuy** [comment, marked as answer]
onedrive can mess with the files, best not to run the mod from one drive.
